src/graph.rs

```diff
@@ -2770,6 +2773,14 @@ struct Builder<'a, 'graph> {
   diagnostics: Vec<BuildDiagnostic>,
 }
 
+#[derive(Error, Debug, Clone)]
+pub enum JsrPackageFormatError {
```

Comment:
I'm not sure if this is the best way to propagate errors. Suggestions welcome.

Reply:
This is good thanks! Minor nitpick: would you be able to move it down to above `validate_jsr_specifier` so it's not between `Builder` and its implementation though?